package.json

```diff
@@ -7,6 +7,7 @@
     "react-native-v1": "lib-rn/react-navigation.js",
     "web": "lib/react-navigation.web.js"
   },
+  "module": "lib/react-navigation.web.js",
   "webpack": "lib/react-navigation.web.js",
```

Comment:
@skevy out website doesn't work without it. probably difference between webpack 1 and 2